<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>595</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Giriş Ekranı</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(54, 54, 54)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>180</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); font-size: 20px</string>
   </property>
   <property name="text">
    <string>Kullanıcı Adı</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255); font-size: 20px</string>
   </property>
   <property name="text">
    <string>Şifre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); font-size: 50px</string>
   </property>
   <property name="text">
    <string> HOŞ GELDİNİZ</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="kullanici_adi_linedit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>210</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); font-size:20px;
border-style: outset;
    border-width: 2px;
    border-radius: 20px;
    border-color:rgb(35, 247, 212);
    min-width: 10em;
    padding: 6px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="giris_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>430</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255); font-size: 20px; background-color:rgb(0, 0, 0);
border-style: outset;
    border-width: 1.5px;
    border-radius: 20px;
    border-color:rgb(35, 247, 212);
    min-width: 10em;
    padding: 6px;</string>
   </property>
   <property name="text">
    <string>Giriş Yap</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="sifre_linedit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>351</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255); font-size:20px;
border-style: outset;
    border-width: 2px;
    border-radius: 20px;
    border-color: rgb(35, 247, 212);
    min-width: 10em;
    padding: 6px;
password:</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="hesap_olustur_btn">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>500</y>
     <width>271</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgb(255, 255, 255); font-size: 20px; background-color:rgb(0, 0, 0);
border-style: outset;
    border-width: 1.5px;
    border-radius: 20px;
    border-color: rgb(35, 247, 212);
    min-width: 10em;
    padding: 6px;</string>
   </property>
   <property name="text">
    <string>Hesap Oluştur</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>120</y>
     <width>271</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(163, 163, 163); font-size: 15px</string>
   </property>
   <property name="text">
    <string>Lütfen devam etmek için giriş yapınız....</string>
   </property>
  </widget>
  <widget class="QLabel" name="error_label">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
